i18n/en.yaml

```
@@ -76,12 +76,30 @@ collections:
   jurisdiction: 
     one: Jurisdiction
     other: Jurisdictions
+  metadatas:
```

“data” is invariable 🙂 
```suggestion
  metadata:
```